.github/workflows/release.yml

```
@@ -90,20 +119,28 @@ jobs:
           # Hack: use FORCE_COLOR so that supports-color@5 returnes true for GitHub CI
           # Remove once `chalk` is bumped to 4.0.
           FORCE_COLOR: true
+          BABEL_8_BREAKING: ${{ needs.check-release-type.outputs.is_babel_9 == 'true' }}
+          USE_ESM: ${{ needs.check-release-type.outputs.is_babel_9 == 'true' }}
```

surely you mean

```suggestion
          BABEL_8_BREAKING: ${{ needs.check-release-type.outputs.is-babel-8 == 'true' }}
          USE_ESM: ${{ needs.check-release-type.outputs.is-babel-8 == 'true' }}
```

? :)

---

Yes thank you :)